client/layout/guided-tours/tours/activity-log-tour.js

```diff
@@ -94,10 +95,7 @@ export const ActivityLogTour = makeTour(
 			style={ { marginLeft: '-18px' } }
 		>
 			<p>
-				{ translate(
-					"In here you'll find each event's options, such as rewinding one by one. " +
-						'Now, go explore!'
-				) }
+				{ translate( "In here you'll find each event's options, such as rewinding one by one." ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'In here you'll find each event's options, such as rewinding one by one. Now, go explore!' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).